### PR TITLE
Transmission_Tests

### PR DESCRIPTION
**Pull request by meadt**
Changed to use a "get_worker_ward_type" function when printing the hospital worker's ward type.

### CODE REVIEW

**Comment by meadt** [comment]
> Remove .idea files. these do not need to be in version control

I'd added them to the gitignore. There's no easy way for me to list files to ignore within CLion, which is annoying, because using gitignore doesn't seem to work like I think it does.

File: src/hospital.c [hunk 1]
```diff
@@ -83,8 +83,9 @@ void set_up_hospital_networks( model *model )
                 for( hcw_idx = 0; hcw_idx < hospital->wards[ward_type][ward_idx].n_worker[DOCTOR]; hcw_idx++ )
                     healthcare_workers[n_healthcare_workers++] = hospital->wards[ward_type][ward_idx].doctors[hcw_idx].pdx;
 
-                for( hcw_idx = 0; hcw_idx < hospital->wards[ward_type][ward_idx].n_worker[NURSE]; hcw_idx++ )
+                for( hcw_idx = 0; hcw_idx < hospital->wards[ward_type][ward_idx].n_worker[NURSE]; hcw_idx++ ) {
```
Comment (kelvin-van-vuuren-crick) [inline src/hospital.c:86]: remove these brackets as it is an unnecessary change

File: src/hospital.c [hunk 2]
```diff
@@ -120,7 +121,7 @@ void build_hospital_networks( model *model, hospital *hospital )
 *  Description: Adds healthcare worker ids to hospital wards
 *  Returns:     void
 ******************************************************************************************/
-void add_healthcare_worker_to_hospital( hospital *hospital, long pdx, int hcw_type )
+void add_healthcare_worker_to_hospital( model *model, hospital *hospital, long pdx, int hcw_type )
```
Comment (kelvin-van-vuuren-crick) [inline src/hospital.c:124]: Why has model being added as an argument here? it is not being used in the function.

File: src/hospital.c [hunk 3]
```diff
@@ -148,9 +149,11 @@ void add_healthcare_worker_to_hospital( hospital *hospital, long pdx, int hcw_ty
     if( hcw_type == DOCTOR )
         initialise_doctor( &(hospital->wards[ward_type][ward_idx].doctors[hospital->wards[ward_type][ward_idx].n_worker[DOCTOR]++]),
                 pdx, hospital->hospital_idx, ward_idx, ward_type);
+
```
Comment (kelvin-van-vuuren-crick) [inline src/hospital.c:152]: Remove these extra spaces

File: src/hospital.h [hunk 4]
```diff
@@ -56,7 +56,7 @@ void transition_to_icu( model *model, individual *indiv );
 void transition_to_mortuary( model *model, individual *indiv );
 void transition_to_discharged( model *model, individual *indiv );
 
-void add_healthcare_worker_to_hospital(hospital *hospital, long pdx, int type);
+void add_healthcare_worker_to_hospital( model *model, hospital *hospital, long pdx, int type);
```
Comment (kelvin-van-vuuren-crick) [inline src/hospital.h:59]: Why has model being added as an argument here? it is not being used in the function.

File: src/model.c [hunk 5]
```diff
@@ -820,7 +820,7 @@ void set_up_healthcare_workers_and_hospitals( model *model)
                 continue;
 
         indiv->worker_type = DOCTOR;
-        add_healthcare_worker_to_hospital( &(model->hospitals[0]), indiv->idx, DOCTOR );
+        add_healthcare_worker_to_hospital( model, &(model->hospitals[0]), indiv->idx, DOCTOR );
```
Comment (kelvin-van-vuuren-crick) [inline src/model.c:823]: Why has model being added as an argument here? it is not being used in the function.

File: src/model.c [hunk 6]
```diff
@@ -838,7 +838,7 @@ void set_up_healthcare_workers_and_hospitals( model *model)
                 continue;
 
         indiv->worker_type = NURSE;
-        add_healthcare_worker_to_hospital( &(model->hospitals[0]), indiv->idx, NURSE );
+        add_healthcare_worker_to_hospital( model, &(model->hospitals[0]), indiv->idx, NURSE );
```
Comment (kelvin-van-vuuren-crick) [inline src/model.c:841]: Why has model being added as an argument here? it is not being used in the function.

File: src/hospital.h (hunk 4 above)
Comment (meadt) [reply src/hospital.h:59]: Removed the model argument.

File: src/model.c (hunk 5 above)
Comment (meadt) [reply src/model.c]: Removed model argument.

File: src/model.c (hunk 6 above)
Comment (meadt) [reply src/model.c]: Removed model argument.
